### ISSUE TITLE
Search does not seem to be working

### ISSUE THREAD

**Issue by nherment** [senecajs/senecajs.org#229]
the search on senecajs.org does not allow much flexibility.
For example I'm looking for timeout management in seneca. If I type `timeout` in the search box. Nothing happens. Typing enter does not work.

**Comment by naomifeehanmoran** [comment]
thanks for raising this @nherment. We will get somebody assigned to this early next week.

**Comment by mcdonnelldean** [comment]
@nherment @naomifeehanmoran I don't think this is a bug so much as a incomplete set of settings. I'll get these updated in the next pass. Thanks for pointing out.
